src/files/selected-actions/SelectedActions.js

```diff
@@ -70,37 +69,12 @@ class SelectedActions extends React.Component {
     force100: false
   }
 
-  componentDidUpdate (prev) {
-    if (this.props.downloadProgress === 100 && prev.downloadProgress !== 100) {
-      this.setState({ force100: true })
-      setTimeout(() => {
-        this.setState({ force100: false })
-      }, 2000)
-    }
-  }
-
   componentDidMount () {
     this.containerRef.current && this.containerRef.current.focus()
   }
 
-  get downloadText () {
-    if (this.state.force100) {
-      return this.props.t('finished')
-    }
-
-    if (!this.props.downloadProgress) {
-      return this.props.t('app:actions.download')
-    }
-
-    if (this.props.downloadProgress === 100) {
-      return this.props.t('finished')
-    }
-
-    return this.props.downloadProgress.toFixed(0) + '%'
-  }
-
   render () {
-    const { t, tReady, animateOnStart, count, size, unselect, remove, share, setPinning, download, downloadProgress, rename, inspect, className, style, isMfs, ...props } = this.props
+    const { t, tReady, animateOnStart, count, size, unselect, remove, share, setPinning, download, rename, inspect, className, style, isMfs, ...props } = this.props
```

Comment:
Can we sort these alphabetically?

Reply:
Unless you know how to do it automatically, please don't make me do it 😅